setup.py

```diff
@@ -24,8 +27,8 @@
     install_requires=[
         "numpy",
         "mrcfile",
-        "keras>=2.1.2,<2.4.0",
-        "csbdeep>=0.6.0,<0.7.0",
+        "csbdeep>=0.7.0,<0.8.0",
+        "packaging",
```

Comment:
No sure, when it was a problem at some point, but it might be just a problem with my cuda 11 setup. I gonna remove it, install everything and see if it works .

Reply:
It works!